_schema.graphql

```diff
@@ -1,19 +1,19 @@
 directive @principalField on FIELD
 
-# Autogenerated input type of AddAssetToConsignmentSubmission
 input AddAssetToConsignmentSubmissionInput {
-  # A unique identifier for the client performing the mutation.
+  # The type of the asset
```

Comment:
That pre-commit hook that runs generated a really large diff to the two schemas, is this expected? If not, any suggestions on getting around that? I was a little surprised to see such a large diff

Reply:
I am guessing `ENABLE_CONSIGNMENTS_STITCHING` env variable is recently set to true on staging but you don't have it in your `.env` file so all the [convection](https://github.com/artsy/convection) stitched schema fields that are already added to master is being removed by `dump-schema`

Reply:
Ah yes, that was exactly it. Thanks for the context!!

Reply:
@jpotts244 - mind updating `.env.example` with this value?

Reply:
will do ^